main.py

```diff
@@ -31,28 +38,41 @@ def read_root():
     # assistant response give a response that will guide the user to find out next steps to take to help their pet
     assistant_response = "Give the user 5 steps to take to help their pet in json format as step_[number] and the action to take as the value with max token as 50"
     
+    gpt_message = [
+        {"role": "system", "content": system_content},
+        {"role": "user", "content": user_content},
+        {"role": "assistant", "content": assistant_response},
+    ]
     
     completion = client.chat.completions.create(
         model="gpt-4-0125-preview",
-        messages=[
-            {"role": "system", "content": system_content},
-            {"role": "user", "content": user_content},
-            {"role": "assistant", "content": assistant_response},
-        ],
+        messages=gpt_message,
         max_tokens=300,
         response_format={
             "type": "json_object"
         },
     )
+    
+    
+    total_tokens = completion.usage.total_tokens
+    #Database needs message, response, tokens
+    
+    # Execute an insert query
+    insert_query = "INSERT INTO token_usage (message, response, tokens) VALUES (%s, %s, %s)"
```

Comment:
why is this insert sql injection in a GET call. this is not good standard and should move into POST CALL

Reply:
lesson for later time.